packages/fxa-settings/src/components/Security/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Security from '.';
+
+const { findByText, findAllByText } = screen;
+
+describe('SecurityPanel', async () => {
+  it('renders "fresh load" SecurityPanel with correct content', () => {
+    render(
+      <Security
+        accountRecoveryKeyEnabled={false}
+        twoFactorAuthEnabled={false}
+      />
+    );
+
+    expect(findByText('Recovery key')).toBeTruthy;
+    expect(findByText('Two-step authentication')).toBeTruthy;
+
+    findAllByText('Not Set').then((result) => {
```

Comment:
Is it possible to use `await` here instead of `.then`?

Reply:
I tried that but couldn't get it working like I wanted, searched around for examples with await in fxa-settings and didn't see any, so went with this